installation/scripts/e2e-testing.sh

```diff
@@ -44,7 +48,11 @@ testcase="${ROOT_PATH}"/../../tests/end-to-end/backup-restore-test/deploy/chart/
 release=$(basename "$testcase")
 
 cleanupHelmE2ERelease "${release}"
-helm install "$testcase" --name "${release}" --namespace end-to-end
+
+ADMIN_EMAIL=$(kubectl get secret admin-user -n kyma-system -o jsonpath="{.data.email}" | base64 -D)
+ADMIN_PASSWORD=$(kubectl get secret admin-user -n kyma-system -o jsonpath="{.data.password}" | base64 -D)
+
+helm install "$testcase" --name "${release}" --namespace end-to-end --set global.ingress.domainName="${DOMAIN}",global.adminEmail="${ADMIN_EMAIL}",global.adminPassword="${ADMIN_PASSWORD}"
```

Comment:
Please use three "--set" arguments instead of a single big one
Please add a function that escapes every value used as override as follows:
- every comma is escaped: `, => \,`
- every equal sign is escaped: `= => \=`